package.json

```diff
@@ -99,7 +99,7 @@
     "webpack-hot-middleware": "^2.13.2"
   },
   "peerDependencies": {
-    "draft-js-plugins-editor": "2.0.0-beta10",
+    "draft-js-plugins-editor": "2.0.0-rc2",
```

Comment:
It's incompatible with `draft-js-plugins-editor@2.0.0-beta10` . compatible if it's larger than `-beta10` (it's `-beta12` , `-rc1` and `-rc2` )